package.json

```diff
@@ -16,18 +16,15 @@
     "vsce": "^1.36.3"
   },
   "scripts": {
-    "postinstall":
```

Comment:
@AnanyaJha, could you please run `node scripts/reformat-with-prettier.js` in your salesforcedx-vscode project? This will undo the automatic formatting changes in this file.